gateway/tests/api/test_v1_program.py

```diff
@@ -216,19 +216,19 @@ def test_upload_custom_image_without_access_to_the_provider(self):
                 "title": "Private function",
                 "dependencies": "[]",
                 "env_vars": env_vars,
-                "image": "icr.io/awesome-namespace/awesome-title",
-                "provider": "default",
+                "image": "docker.io/awesome-namespace/awesome-title",
+                "provider": "ibm",
```

Comment:
wouldn't ibm images always been in icr.io ? or is there an issue with using icr.io in the tests (looks like we're moving all the tests to docker.io, so maybe that's it?)